taqueria-protocol/Config.ts

```diff
@@ -27,13 +41,15 @@ const sandboxMap = z
         {description: "Sandbox configurations"}
     )
     .optional()
+    
+const storageSchema = z.unknown().optional()
 
 const environmentMap = z
     .record(
         z.union([
             Environment.schemas.schema,
-            z.string({description: "config.environment"})
-            .min(1, "Default environment must reference the name of an existing environment.")
+            z.string().min(1, "Default environment must reference the name of an existing environment."),
```

Comment:
Are we able to use i18n inside a zod function like this?

Reply:
We are :)

I've been leaving out i18n till we have someone able to add some translations.